### ISSUE TITLE
Notebook "Multiple_Watersheds_Simulation" fails with a "broken pipe" error

### ISSUE THREAD

**Issue by richardarsenault** [Ouranosinc/raven#361]
When running the second cell, I get this error:

```
---------------------------------------------------------------------------
ServiceException                          Traceback (most recent call last)
<ipython-input-4-5fbc459fe23f> in <module>
     20 
     21 # Launch the WPS to get the multi-model results.  Note the "gr4jcn" and "hmets" keys.
---> 22 resp=wps.raven_gr4j_cemaneige(ts=str(ts),params=params, **config)
     23 
     24 # And get the response

</opt/conda/envs/birdy/lib/python3.7/site-packages/birdy/client/base.py-240> in raven_gr4j_cemaneige(self, ts, latitude, longitude, elevation, nc_spec, params, start_date, end_date, nc_index, duration, run_name, name, area, evaporation, rain_snow_fraction, rvc, output_formats)

/opt/conda/envs/birdy/lib/python3.7/site-packages/birdy/client/base.py in _execute(self, pid, **kwargs)
    345         try:
    346             wps_response = self._wps.execute(
--> 347                 pid, inputs=wps_inputs, output=wps_outputs, mode=mode
    348             )
    349 

/opt/conda/envs/birdy/lib/python3.7/site-packages/owslib/wps.py in execute(self, identifier, inputs, output, mode, lineage, request, response)
    357         # submit the request to the live server
    358         if response is None:
--> 359             response = execution.submitRequest(request)
    360         else:
    361             response = etree.fromstring(response)

/opt/conda/envs/birdy/lib/python3.7/site-packages/owslib/wps.py in submitRequest(self, request)
    910         reader = WPSExecuteReader(verbose=self.verbose, timeout=self.timeout, auth=self.auth)
    911         response = reader.readFromUrl(
--> 912             self.url, request, method='Post', headers=self.headers)
    913         self.response = response
    914         return response

/opt/conda/envs/birdy/lib/python3.7/site-packages/owslib/wps.py in readFromUrl(self, url, data, method, username, password, headers, verify, cert)
    601 
    602         return self._readFromUrl(url, data, self.timeout, method, username=username, password=password,
--> 603                                  headers=headers, verify=verify, cert=cert)
    604 
    605 

/opt/conda/envs/birdy/lib/python3.7/site-packages/owslib/wps.py in _readFromUrl(self, url, data, timeout, method, username, password, headers, verify, cert)
    513             u = openURL(url, data, method='Post',
    514                         username=self.auth.username, password=self.auth.password,
--> 515                         headers=headers, verify=self.auth.verify, cert=self.auth.cert, timeout=timeout)
    516             return etree.fromstring(u.read())
    517 

/opt/conda/envs/birdy/lib/python3.7/site-packages/owslib/util.py in openURL(url_base, data, method, cookies, username, password, timeout, headers, verify, cert, auth)
    204 
    205     if req.status_code in [400, 401]:
--> 206         raise ServiceException(req.text)
    207 
    208     if req.status_code in [404, 500, 502, 503, 504]:    # add more if needed

ServiceException: <?xml version="1.0" encoding="utf-8"?>
<ExceptionReport version="1.0.0"
    xmlns="http://www.opengis.net/ows/1.1"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd">
    <Exception exceptionCode="NoApplicableCode" locator="NotAcceptable">
        <ExceptionText>Request failed: (&#x27;Connection aborted.&#x27;, BrokenPipeError(32, &#x27;Broken pipe&#x27;))</ExceptionText>
    </Exception>
</ExceptionReport>
```

**Comment by Zeitsperre** [comment]
Related? https://github.com/bird-house/finch/issues/98

**Comment by Zeitsperre** [comment]
Fixed by #374 ?
